### PR TITLE
🐦[FSharp] Falco 1.2.1

### PR DESCRIPTION
**Pull request by pimbrouwers**
Some small tweaks to the Program.fs and update to Falco 1.2.1

### CODE REVIEW

**Comment by pimbrouwers** [comment]
I was curious if response compression/caching are enabled for any of these frameworks implicitly? This is would have a significant impact on performance, and is not enabled by default is ASP.NET core.

File: fsharp/falco/web.fsproj [hunk 1]
```diff
@@ -4,8 +4,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Program.fs" />
-  </ItemGroup>
+  </ItemGroup>  
   <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.7.2" />
```
Comment (waghanza) [inline fsharp/falco/web.fsproj:9]: The `F#` version is not provided by the container ?

**Comment by pimbrouwers** [comment]
Fixed. Sorry about that.

**Comment by waghanza** [comment]
@pimbrouwers It was probably not a bug, I mean I don't know, it's up to you

/cc @panesofglass

**Comment by waghanza** [comment]
After checking, I think you are right.

Into the container, I run `fhsarpi` and I have
```fhsarp
Microsoft (R) F# Interactive version 10.2.3 for F# 4.5
Copyright (c) Microsoft Corporation. All Rights Reserved.
```

**Comment by pimbrouwers** [comment]
Apologies. The two github threads had me all mixed up. 

It is typical for f# projects to reference an explicit language version in the project. 

So long as the docker container is up to date it will possess the 4.7.2 version.

**Comment by waghanza** [comment]
> It is typical for f# projects to reference an explicit language version in the project.

OK, I take this :heart:

**Pull request closed without merging**